hwilib/devices/bitbox02_lib/bitbox02/bitbox02.py

```diff
@@ -32,20 +32,25 @@
 
 from .secp256k1 import antiklepto_host_commit, antiklepto_verify
 
-from ..communication.generated import hww_pb2 as hww
```

Comment:
Split the bitbox02_lib update into a separate commit, mentioning it is updating to v6.3.0. Please re-do this from this tag: 

https://github.com/digitalbitbox/bitbox02-firmware/tree/py-bitbox02-6.3.0

Reply:
Good idea. Doing that now.